src/test/java/digital/slovensko/autogram/AutogramTests.java

```diff
@@ -28,14 +30,16 @@
 import static org.mockito.Mockito.*;
 
 class AutogramTests {
-    @Test
-    void testSignHappyScenario() {
+    @ParameterizedTest
+    @MethodSource({ "digital.slovensko.autogram.TestMethodSources#generalAgendaProvider",
+            "digital.slovensko.autogram.TestMethodSources#validOtherDocumentsProvider",
+            "digital.slovensko.autogram.TestMethodSources#validXadesDocumentsProvider"})
```

Comment:
Tento zapis nepoznam, co to robi?

Reply:
Do @MethodSource idú metódy, ktoré vracajú listy vstupných argumentov pre test. Čiže netreba kopírovať jeden test N-krát pre N súborov na podpis, ale dať tam metódu, ktorá vráti N súborov a test sa tak spustí N-krát. A viacero metód iba spojí vstupy do dlhšieho listu N+M+K.